Dockerfile

```diff
@@ -1,7 +1,7 @@
 # To run: docker run -d --name=dataportal -p 80:80 quay.io/cdis/data-portal
 # To check running container: docker exec -it dataportal /bin/bash
 
-FROM quay.io/cdis/ubuntu:18.04
+FROM quay.io/cdis/ubuntu:20.04
```

Comment:
Just curious, do we have a cdis `node` docker image?

Reply:
I think we only have one `12.10.0-alpine` node image